.travis.yml

```diff
@@ -82,6 +82,13 @@ jobs:
         - BLOCKCHAIN_TYPE='tester'
         - TEST='raiden/tests/unit raiden/tests/integration/contracts'
 
+    - stage: test
+      <<: *job-template-integration
+      env:
+        - TEST='raiden/tests/integration --ignore=raiden/tests/integration/transfer --ignore=raiden/tests/integration/long_running --ignore=raiden/tests/integration/api --ignore=raiden/tests/integration/contracts --ignore=raiden/tests/integration/cli'
+        - TRANSPORT_OPTIONS='--transport=matrix --local-matrix=${HOME}/.bin/run_synapse.sh'
+        - RUN_SYNAPSE=1
+
```

Comment:
Do we want to always run the basic integration tests in the 'test' stage?

Reply:
It's in the WIP commit, just a small hack to be able to run only this job (which was the one with flaky tests), it also sets the raiden-libs to my personal fork, and is intended to be dropped before merge.